app/components/search/SearchEntry.jsx

```diff
@@ -7,6 +7,19 @@ import { RelativeOpeningTime } from '../listing/RelativeOpeningTime';
 import './SearchEntry.scss';
 
 class SearchEntry extends Component {
+  renderAddressMetadata() {
+    const { hit } = this.props;
+    const { addresses } = hit;
+    if (addresses && addresses.length) {
```

Comment:
if you make ln 12 use a default value for destructuring (i.e. `const { addresses = [] } = hit;`), then you could un-nest this if statement and just have it as three cases instead of two, i.e.

```
if (address.length === 0) {
    return <span>No address found</span>;
} else if (addresses.length > 1) {
    return <span>Multiple locations</span>;
} else if (addresses[0].address_1) {
    return <span>{addresses[0].address_1}</span>;
}

Reply:
ohh good call. Updated!